components/lib/progressbar/primereact.code-workspace

```diff
@@ -0,0 +1,30 @@
+{
+    "folders": [
+        {
+            "name": "primereact",
```

Comment:
I htink this file was an accidental commit?  Should be add `*.code-workspace` to `.gitignores`?